cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/CloudInstance.java

```diff
@@ -54,7 +54,7 @@ public CloudInstance(@JsonProperty("instanceId") String instanceId,
             @JsonProperty("authentication") InstanceAuthentication authentication,
             @JsonProperty("subnetId") String subnetId,
             @JsonProperty("availabilityZone") String availabilityZone,
-            @JsonProperty("params") Map<String, Object> params) {
+            @JsonProperty("parameters") Map<String, Object> params) {
```

Comment:
was this intentional? won't this cause any issue? do we store this anywhere in the db? if yes is this backward comaptible?

Reply:
it is intentional as below. The field and getter name is `parameters`

Reply:
ok